openlibrary/plugins/openlibrary/js/partner_ol_lib.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @param {string} container
+ */
+function getIsbnToElementMap(container) {
+    const reISBN = /((978)?[0-9][0-9]{10}[0-9xX])|((978)?[0-9]{9}[0-9Xx])/;
```

Comment:
This regex is wrong :P 
```suggestion
    const reISBN = /(978)?[0-9]{9}[0-9X]/i;
```